clang/include/clang/CConv/ProgramInfo.h

```diff
@@ -117,6 +117,11 @@ class ProgramInfo : public ProgramVariableAdder {
   void constrainWildIfMacro(ConstraintVariable *CV, SourceLocation Location,
                             PersistentSourceLoc *PSL = nullptr);
 
+  std::map<PersistentSourceLoc, std::pair<CVarSet, bool>> typedefVars;
+
+  void checkTypedef(const clang::Type*, clang::ASTContext&,
+                    clang::DeclaratorDecl*, PVConstraint*);
```

Comment:
I can't guess what this function does from the name. What does it mean to "check" a typdef?